css/features.scss

```diff
@@ -19,10 +11,12 @@
     display: flex;
     flex-direction: column;
     margin: 50px 0;
+
     .features__section__icon {
```

Comment:
According to BEM methology you can't have element inside another element. Consider to convert parent element to standalone block in this case or just rethink block hierarchy.